pkg/cmd/server/api/latest/helpers.go

```diff
@@ -158,3 +160,31 @@ func IsAdmissionPluginActivated(reader io.Reader, defaultValue bool) (bool, erro
 
 	return !activationConfig.Disable, nil
 }
+
+func ConvertOpenshiftAdmissionConfigToKubeAdmissionConfig(in map[string]configapi.AdmissionPluginConfig) (*apiserver.AdmissionConfiguration, error) {
```

Comment:
If you want to have it in helpers.go, wouldn't be better to have it only for one? and then call on multiple plugins where needed?

Reply:
> If you want to have it in helpers.go, wouldn't be better to have it only for one? and then call on multiple plugins where needed?

Good question. Ordinarily I'd agree with you, but this actually matches a type in our server config here: https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/types.go#L1504, so when we're done, it will operate directly on the config instance variable.